src/main/resources/application-prod.yml

```
@@ -30,6 +30,8 @@ spring:
     jdbc:
       initialize-schema: never
 
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)